src/main.ts

```diff
@@ -1 +1,3 @@
-console.log("Hello world!")
+var yourName = "John";
```

Comment:
Codacy has a fix for the issue: [Unexpected var, use let or const instead.](https://app.codacy.com/gh/lolgab/eslint-example/pullRequest?prid=6956677)
```suggestion
let yourName = "John";
```